library/ceph_disks.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+DOCUMENTATION = '''
+---
+author: Siva Mullapudi
+module: ceph_disks
+short_description: Outputs a list of valid devices
+description:
+    - Outputs a list of devices based on the osd type.
+options: none
+'''
+
+EXAMPLES = '''
+# Assuming a journal_collocation osd type
+- ceph_disks:
+  register: ssds
+'''
+
+import os
+import re
+import subprocess
+import json
+from subprocess import Popen, PIPE
+
+def run_command(cmd):
+    process = Popen(cmd, stdout=PIPE, stderr=PIPE, shell=True)
+    stdout, stderr = process.communicate()
+    return stdout.rstrip('\n')
```

Comment:
Ansible module shared code has convenience methods for running commands. We should be using those instead of creating our own.